config/settings.yml

```
@@ -10,7 +10,13 @@
     :service:
       :read_timeout: 1.hour
       :open_timeout: 1.minute
-    :blacklisted_event_names: []
+    :blacklisted_event_names:
+    - UNASSIGNED
+    - USER_REMOVE_VG
+    - USER_REMOVE_VG_FAILED
+    - USER_VDC_LOGIN
+    - USER_VDC_LOGOUT
+    - USER_VDC_LOGIN_FAILED
```

Not important, but alpha-sort

```suggestion
    - USER_VDC_LOGIN
    - USER_VDC_LOGIN_FAILED
    - USER_VDC_LOGOUT
```